### PR TITLE
Clean-up parabolic bowl convergence plot

### PR DESCRIPTION
**Pull request by cbegeman**
Invert the x-axis for the parabolic bowl convergence plot

### CODE REVIEW

**Comment by cbegeman** [comment]
### Testing

The standard, ramp parabolic bowl test case has been run on chrys with intel, impi compiler. 

image

File: compass/ocean/tests/parabolic_bowl/viz/__init__.py [hunk 1]
```diff
@@ -231,19 +232,20 @@ def rmse_plots(self):
...
+        for i in range(len(resolutions) - 1):
             rmse_1st_order[i + 1] = rmse_1st_order[i] / 2.0
 
-        ax.loglog(self.resolutions, rmse_1st_order,
+        ax.loglog(resolutions, rmse_1st_order,
                   linestyle='-', color='k', alpha=.25, label='1st order')
 
         ax.set_xlabel('Cell size (km)')
         ax.set_ylabel('RMSE (m)')
+        ax.invert_xaxis()
```
Comment (sbrus89) [inline compass/ocean/tests/parabolic_bowl/viz/__init__.py:248]: @cbegeman, this is the only real substantive change here, correct? The `self.resolutions` replacements are a nice cleanup, though. Thanks for fixing this!

Comment (cbegeman) [reply compass/ocean/tests/parabolic_bowl/viz/__init__.py:248]: Yep. I originally thought we also needed to sort `resolutions` but that ended up not being necessary.

**Review by sbrus89 — APPROVED** [review]
Approved by inspection. Thanks for fixing this @cbegeman.

**Pull request merged**